imports/blocks/add-schedule/phrases/index.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+type ITertiaryPhrase = {
+  additionalClasses?: string,
+  text?: string,
+};
+
+const TertiaryPhrase = ({
```

Comment:
This brings up an interesting idea of creating type style components for use. Maybe with https://github.com/styled-components/styled-components?